frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -557,68 +548,117 @@ function removeForestClientFromList(index: number) {
                                 </Button>
                             </div>
                         </div>
+                    </StepContainer>
+                </div>
+
+                <div v-if="forestClientData.length > 0">
+                    <div class="px-0">
+                        <ForestClientCard
```

Comment:
I feel we can include this part in the 3rd StepContainer as well?

Reply:
That makes more sense, thank you! I have added it.